template/webpack.config.js

```diff
@@ -2,17 +2,18 @@ const path = require('path')
 
 module.exports = {
   entry: './main.js',
-  mode: process.env.WEBPACK_SERVE ? 'development' : 'production',
+  devServer: {
+    contentBase: './dist',
+    historyApiFallback: true,
+    hot: true,
+    inline: true
+  },
+  mode: process.env.DEVELOPMENT ? 'development' : 'production',
```

Comment:
I'm not aware of an environment variable that would tell us that the dev-server was being used, so we have to set one up in the start script